### PR TITLE
updated files in branch1

### PR DESCRIPTION
**Pull request by Rami-Ilaiwi**
Please review.

### CODE REVIEW

File: project1/src/components/Slug.tsx [hunk 1]
```diff
@@ -167,13 +166,17 @@ class Slug extends React.Component<RouteComponentProps<{ slug: string }>> {
           <Grid item></Grid>
           {/* style={{ marginLeft: "25%", marginTop: "2%" }} */}
           <Grid item>
-            <hr />
+            <hr style={{ width: "1000px" }} />
+
+            {localStorage.getItem("userToken") ? (
```
Comment (Abdallatif) [inline project1/src/components/Slug.tsx:171]: move localStorage.getItem to the top of the function to keep the jsx clean

File: project1/src/components/Comments.tsx [hunk 2]
```diff
@@ -18,34 +25,55 @@ interface commentsProps {
...
-            )}
-          </span>
-          {/* <span className="mod-options ng-hide" ng-show="$ctrl.canModify">
-            <i className="ion-trash-a" ng-click="$ctrl.deleteCb()"></i>
-          </span> */}
-        </div>
+        <Card className={classes.card}>
+          <CardContent>
+            <Typography gutterBottom variant="h5" component="h2">
+              <a className="comment-author" href="#/@yiwei">
```
Comment (Abdallatif) [inline project1/src/components/Comments.tsx:42]: what does the href here mean?

File: project1/src/components/Comments.tsx [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
 import moment from "moment";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
 
 interface commentItem {
```
Comment (Abdallatif) [inline project1/src/components/Comments.tsx:11]: let's keep all the types in PascalCase
```suggestion
interface CommentItem {
```

File: project1/src/components/Comments.tsx (hunk 2 above)
Comment (Rami-Ilaiwi) [reply project1/src/components/Comments.tsx:42]: I will edit it later, this link should go to the article's author profile.

**Pull request merged**